### PR TITLE
docs(community): adding zodios usage and example

### PR DESCRIPTION
**Pull request by enyelsequeira**
# Changes

I think zodios is a great example that could be included in the `Community Resources`. It is easy to use and all with typesafety in mind.

### CODE REVIEW

**Review by TkDodo — CHANGES_REQUESTED** [review]
you need to add an entry to `config.json`:

https://github.com/TanStack/query/blob/765f6196e900c1dd826b040773fcf97839ca60d0/docs/config.json#L190-L208

**Comment by TkDodo** [comment]
@ecyrbe FYI :)

**Comment by enyelsequeira** [comment]
> you need to add an entry to `config.json`:
> 
> https://github.com/TanStack/query/blob/765f6196e900c1dd826b040773fcf97839ca60d0/docs/config.json#L190-L208

Understood it has been added and changed id of .md

**Comment by ecyrbe** [comment]
Hello, i don't think this is the way to integrate zodios in tanstack query docs.

Because Zodios can change interface, have breaking changes, etc and tanstack query will have a hard time to maintain these docs in sync.

A simple link to some Zodios docs would be enough in my point of view.

What do you think ?

**Comment by enyelsequeira** [comment]
> Hello, i don't think this is the way to integrate zodios in tanstack query docs.
> 
> Because Zodios can change interface, have breaking changes, etc and tanstack query will have a hard time to maintain these docs in sync.
> 
> A simple link to some Zodios docs would be enough in my point of view.
> 
> What do you think ?

Hey, so I think every resource out there will have breaking changes. When the moment comes someone would just have to update the documentation for it. I think I’ve seen some other pull request updating docs as something changes as well. Just my thoughts though.

File: docs/config.json [hunk 1]
```diff
@@ -204,6 +204,10 @@
             {
               "label": "Angular Query",
               "to": "react/community/angular-query"
+            },
+            {
+              "label": "Zodios",
+              "to": "react/community/zodios-by-ecrybe"
```
Comment (ecyrbe) [inline docs/config.json:210]:
```suggestion
              "to": "react/community/zodios-by-ecyrbe"
```

File: docs/react/community/zodios.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
+---
+id: zodios-by-ecrybe
```
Comment (ecyrbe) [inline docs/react/community/zodios.md:2]:
```suggestion
id: zodios-by-ecyrbe
```

**Review by ecyrbe — COMMENTED** [review]
Some minor typo 😁

File: docs/config.json (hunk 1 above)
Comment (enyelsequeira) [reply docs/config.json]: oops, my mistake 😅, spelling has been fixed :)

**Comment by TkDodo** [comment]
I've discussed with @ecyrbe and we've decided to go a little different approach: We'll draft one "Awesome React Query Collection" (similar to how [trpc](https://trpc.io/docs/community/awesome-trpc) does it) page where we'll list all community packages with links to docs, github and also when this might be a good fit. This seems better than a separate page for each, with a "quickstart" guide that might get outdated fast

**Pull request closed without merging**